src/unix/stream.c

```diff
@@ -1625,6 +1625,21 @@ void uv__stream_close(uv_stream_t* handle) {
 }
 
 
+/* Have stream block and then synchronously flush queued writes.
+ * This function works without an event loop.
+ * Intended to be used just prior to exit().
+ * Returns 0 on success, non-zero on failure.
+ */
+int uv_flush_sync(uv_stream_t* stream) {
+  int rc = uv_stream_set_blocking(stream, 1);
```

Comment:
We could, but it sounds like that won't help get this anywhere.

This was an example patch cooked up that Node.js may need in some form.